components/family-studies/api/src/main/java/org/phenotips/studies/family/internal/PhenotipsFamily.java

```diff
@@ -193,6 +193,13 @@ public String getURL(String actions)
         return getXDocument().getURL(actions, getXContext());
     }
 
+    @Override
+    public boolean containsSensitiveData()
+    {
+        BaseObject familyObject = getXDocument().getXObject(Family.CLASS_REFERENCE);
+        return (familyObject.getIntValue(WARNING) != 0);
```

Comment:
Just wondering why not `== 1`.

Reply:
Just used the same check already in the code. I'm not 100% sure off the top of my head what xwiki uses for booleans, apparently this check used to work well so reused it. If you know it is 1 i agree `== 1` looks better

Reply:
Booleans can have 3 values: 0 means false, 1 means true, -1 means not specified. The warning should only be displayed if it is set to 1, since "unspecified" shouldn't imply there's a warning.

So, change the check to `== 1`.

Reply:
@allasm Added this change to #2801 .